systems/setonix/settings.sh

```diff
@@ -133,6 +133,7 @@ quay.io/pawsey/openfoam:v2012
 quay.io/pawsey/openfoam:v2006
 quay.io/pawsey/openfoam:v1912
 quay.io/pawsey/openfoam:v1812
+quay.io/pawsey/openfoam-org:10
```

Comment:
@AlexisEspinosaGayosso , for this work, you will need to submit a PR to https://github.com/singularityhub/shpc-registry, in order to add a YAML file for version 10, `10.yaml`, containing the list of executables, inside https://github.com/singularityhub/shpc-registry/tree/main/quay.io/pawsey/openfoam-org/aliases

Reply:
The list of aliases has been updated and the PR to the singularityhub has been requested.